lxd/main_callhook.go

```diff
@@ -115,7 +311,6 @@ func (c *cmdCallhook) Run(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		break
```

Comment:
why is this removed?

Reply:
this is removed because this is a useless break case in this context: In a select statement, only one case is executed when a channel operation succeeds so once a case is selected and its associated code block is executed, the select statement automatically terminates (there's no fall-through behavior like in a switch statement). So when the response case is selected (i.e., a value is received from the channel), the code inside that case is executed, and then the select statement naturally completes. The break statement here doesn't break out of any additional loop or switch statement; it's just breaking out of the select, which would happen anyway.

Reply:
Yeah I see, in that case it should be a separate commit as its not related to iGPU at all. Then it can be backported.

Reply:
fixed